packages/sdks/web-component/test/helpers/index.test.ts

```diff
@@ -59,8 +59,12 @@ describe('helpers', () => {
       writable: true,
       value: new URL('http://localhost'),
     });
-    window.location.search = `?${URL_RUN_IDS_PARAM_NAME}=8_9`;
-    expect(getRunIdsFromUrl()).toEqual({ executionId: '8', stepId: '9' });
+    window.location.search = `?${URL_RUN_IDS_PARAM_NAME}=8|#|a_9`;
+    expect(getRunIdsFromUrl('8')).toEqual({
```

Comment:
let's also add test in case of empty flow id, and cases with wrong location.search query param

Reply:
done